package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "coin-hive",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
let the owner do the patching please, also file mode changed...

Reply:
I didn't change that either, That was updated by @cazala